modules/wif-configuration-update.adoc

```diff
@@ -30,6 +30,10 @@ When you update a wif-config or create a new one, ensure your {cluster-manager}
 Error: failed to create wif-config: failed to create wif-config: status is 400, identifier is '400', code is 'CLUSTERS-MGMT-400', at '2025-10-06T15:18:37Z' and operation identifier is 'f9551d63-a58a-4e3c-b847-5f99ba1b0b74': Client version is out of date for WIF operations. Please update from vOCM-CLI/1.0.7 to v1.0.8 and try again.
 ----
 
+You can also update an existing {product-title} cluster that is already using WIF by adding a dedicated project to manage workload identity pools and providers using the `--federated-project` flag. This best-practice model separates the workload identity pools and providers into a dedicated, centralized {GCP} project.
+
+When you update the configuration using the `--federated-project` flag, the federated identity pool moves to the new federated project you specify, while the existing IAM service accounts and custom roles remain in the original cluster-associated project.
```

Comment:
@shreyansvm @AedinC  I believe a section should be included to inform the users that they may manually remove the identity pool from the original project once they have moved the pool to the federated project.